parity/dir.rs

```diff
@@ -73,6 +73,16 @@ impl Directories {
 		dir.push("ipc");
 		dir
 	}
+
+	/// Get user defaults path
+	pub fn user_defaults_path(&self, fork_name: &Option<String>) -> PathBuf {
```

Comment:
I preferred do be consistent with database. Currently it's using fork_name [here](https://github.com/ethcore/parity/blob/master/parity/dir.rs#L58) and [here](https://github.com/ethcore/parity/pull/2014/files#diff-9745e307e54d3e94e4fbba3150efaf8cL108)